contracts/packages/bindings/src/bin/schema.rs

```diff
@@ -0,0 +1,19 @@
+use std::env::current_dir;
+use std::fs::create_dir_all;
+
+use cosmwasm_schema::{export_schema_with_title, remove_schemas, schema_for};
+
+use bindings::WardenProtocolMsg;
+
+fn main() {
```

Comment:
Should we push the generated schema to the repo or not? What do you think?

Reply:
That's a good question! I wouldn't want to do this manually, because there's always a chance of forgetting to do it. It might be a good idea to automate this step.
Anyway, I'll update the readme file to make it clear how to do it manually.